debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -89,11 +84,17 @@
     private final boolean includeUnknownDatatypes;
 
     private final TypeRegistry typeRegistry;
+    private final HStoreMode hStoreMode;
 
-    protected PostgresValueConverter(DecimalMode decimalMode, TemporalPrecisionMode temporalPrecisionMode, ZoneOffset defaultOffset, BigIntUnsignedMode bigIntUnsignedMode, boolean includeUnknownDatatypes, TypeRegistry typeRegistry) {
+    public enum HStoreMode{
```

Comment:
Why is there this second enum? Can't we just use `HStoreHandlingMode` everywhere?

Reply:
I took Decimal as reference , the way it is handling configuration option from the user . However I think having one enum will centralise the change , rather then changing at multiple places if we need to add a new config option for hstore handling.